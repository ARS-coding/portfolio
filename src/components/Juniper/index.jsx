import React from 'react';
import './index.css';
import { Col, Row } from 'react-bootstrap';
import { VerticalTimelineElement } from 'react-vertical-timeline-component';

const Juniper = () => {
  return (
    <VerticalTimelineElement
      className='vertical-timeline-element--work'
      date={'Nov 2020'}
      dateClassName={'date'}
      contentStyle={{
        background: 'white',
        color: 'white',
      }}
      contentArrowStyle={{
        borderRight: '7px solid  white',
      }}
      iconStyle={{
        background: 'white',
      }}
      icon={<i className='fas fa-project-diagram projectIcon'></i>}
    >
      <div className='homeText projectContent'>
        <h4>Juniper</h4>
        <small>Team Project</small> <br />
        <a
          href='https://github.com/ReCoded-Org/capstone-istanbul-juniper'
          target='_blank'
          rel='noreferrer'
        >
          Github
        </a>{' '}
        |{' '}
        <a
          href='https://juniper-ist.netlify.app/'
          target='_blank'
          rel='noreferrer'
        >
          Demo
        </a>
        <p className='projectText'>
          <strong>Goal:</strong> teaching middle-eastern youth about
          environmental problems through games.
        </p>
        <br />
        <Row>
          <Col xs={12}>
            <h5>My Impact</h5>
            <ul>
              <li>Built signature game(memory game) of the project</li>
              <li>
                Elevated my teammates' work by organizing off-hour online
                meetings
              </li>
              <li>Became problem solver of the team</li>
            </ul>
          </Col>
          <Col xs={12} md={6}>
            <h5>Skills Applied</h5>
            <ul>
              <li>HTML</li>
              <li>CSS</li>
              <li>JavaScript</li>
              <li>React</li>
              <li>Git and Github</li>
              <li>AGILE Methodologies</li>
              <li>Teamwork</li>
              <li>Firebase</li>
              <li>Ant Design</li>
              <li>Responsive Design</li>
              <li>Figma</li>
              <li>Testing (Jest)</li>
              <li>Routing</li>
              <li>i18n</li>
            </ul>
          </Col>
          <Col xs={12} md={6}>
            <h5>My Responsibilities</h5>
            <ul>
              <li>Developing a memory game</li>
              <li>Code reviewing</li>
              <li>Bug bashing</li>
              <li>Developing games page</li>
              <li>Scrum master for 1 week</li>
              <li>Developing twitter section of homepage</li>
            </ul>
          </Col>
        </Row>
      </div>
    </VerticalTimelineElement>
  );
};

export default Juniper;
