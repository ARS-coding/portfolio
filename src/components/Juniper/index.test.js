import React from 'react';
import renderer from 'react-test-renderer';
import Juniper from '.';

it('Juniper renders correctly', () => {
  const tree = renderer.create(<Juniper />).toJSON();
  expect(tree).toMatchSnapshot();
});
