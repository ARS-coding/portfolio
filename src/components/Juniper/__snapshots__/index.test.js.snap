// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Juniper renders correctly 1`] = `
<div
  className="vertical-timeline-element--work vertical-timeline-element"
  id=""
  style={null}
>
  <span
    className="vertical-timeline-element-icon is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
      }
    }
  >
    <i
      className="fas fa-project-diagram projectIcon"
    />
  </span>
  <div
    className="vertical-timeline-element-content is-hidden"
    onClick={null}
    style={
      Object {
        "background": "white",
        "color": "white",
      }
    }
  >
    <div
      className="vertical-timeline-element-content-arrow"
      style={
        Object {
          "borderRight": "7px solid  white",
        }
      }
    />
    <div
      className="homeText projectContent"
    >
      <h4>
        Juniper
      </h4>
      <a
        href="https://github.com/ReCoded-Org/capstone-istanbul-juniper"
        rel="noreferrer"
        target="_blank"
      >
        Github
      </a>
       
      |
       
      <a
        href="https://juniper-ist.netlify.app/"
        rel="noreferrer"
        target="_blank"
      >
        Demo
      </a>
      <p>
        <strong>
          Goal:
        </strong>
         teaching middle-eastern youth about environmental problems through games.
      </p>
      <br />
      <div
        className="row"
      >
        <div
          className="col-md-6 col-12"
        >
          <h5>
            Skills Applied
          </h5>
          <ul>
            <li>
              HTML
            </li>
            <li>
              CSS
            </li>
            <li>
              JavaScript
            </li>
            <li>
              React
            </li>
            <li>
              Git and Github
            </li>
            <li>
              AGILE Methodologies
            </li>
            <li>
              Teamwork
            </li>
            <li>
              Firebase
            </li>
            <li>
              Ant Design
            </li>
            <li>
              Responsive Design
            </li>
            <li>
              Figma
            </li>
            <li>
              Testing (Jest)
            </li>
            <li>
              Routing
            </li>
            <li>
              i18n
            </li>
          </ul>
        </div>
        <div
          className="col-md-6 col-12"
        >
          <h5>
            My Responsibilities
          </h5>
          <ul>
            <li>
              Developing a memory game
            </li>
            <li>
              Code reviewing
            </li>
            <li>
              Bug bashing
            </li>
            <li>
              Developing games page
            </li>
            <li>
              Scrum master for 1 week
            </li>
            <li>
              Helping colleagues whenever needed
            </li>
            <li>
              Developing twitter section of homepage
            </li>
          </ul>
        </div>
      </div>
    </div>
    <span
      className="date vertical-timeline-element-date"
    >
      Nov 2020
    </span>
  </div>
</div>
`;
